### ISSUE TITLE
Can’t find Roborock integration in HACs

### ISSUE THREAD

**Issue by BoHome88** [humbertogontijo/homeassistant-roborock#382]
I was trying to install the roborock integration through HACs but I don’t seem to be able to find it I. HACs. Has it been removed?

**Comment by Lash-L** [comment]
You have to add a custom repository to hacs and enter in the url for this repository

**Comment by BoHome88** [comment]
> 

Thanks for the advice Lash-L. Do you mind showing me which url I should be using?

**Comment by Lash-L** [comment]
https://github.com/humbertogontijo/homeassistant-roborock

**Comment by BoHome88** [comment]
> https://github.com/humbertogontijo/homeassistant-roborock

Many thanks for that. I successfully installed the integrations but I get an error message on after I logged into the integration. 
![IMG_0667](https://github.com/humbertogontijo/homeassistant-roborock/assets/117252014/3a4081d4-4f31-478d-8bb8-d5b24ccf04ab)

**Comment by Lash-L** [comment]
Do you have a device on the same local network?

**Comment by The-Croz** [comment]
I'm getting this now too on 1.0.6 running 2023.6.2. Both of my vacuums are connected to the network and work fine on the Roborock app. Here is all it gives me in the log:
```
Logger: homeassistant.config_entries
Source: config_entries.py:947
First occurred: 3:47:43 PM (1 occurrences)
Last logged: 3:47:43 PM

Config entry 'jhc727@gmail.com' for roborock integration not ready yet: There are no devices that can currently be reached.; Retrying in background
```

**Comment by BoHome88** [comment]
> 

Yes. The vacuum and HA are on the same network. My roborock app gives a notification saying I’m logged on another device when I log into the roborock integration on HA. But I still can’t get the integration to connect to my device.

**Comment by q1t1** [comment]
> I'm getting this now too on 1.0.6 running 2023.6.2. Both of my vacuums are connected to the network and work fine on the Roborock app. Here is all it gives me in the log:
> 
> ```
> Logger: homeassistant.config_entries
> Source: config_entries.py:947
> First occurred: 3:47:43 PM (1 occurrences)
> Last logged: 3:47:43 PM
> 
> Config entry 'EMAIL ADDRESS' for roborock integration not ready yet: There are no devices that can currently be reached.; Retrying in background
> ```

I had the same error after rolling back from 1.0.7.

I uninstall / reinstall the integration in HACS, reboot, and the configuration was good after that.

**Comment by The-Croz** [comment]
> > I'm getting this now too on 1.0.6 running 2023.6.2. Both of my vacuums are connected to the network and work fine on the Roborock app. Here is all it gives me in the log:
> > ```
> > Logger: homeassistant.config_entries
> > Source: config_entries.py:947
> > First occurred: 3:47:43 PM (1 occurrences)
> > Last logged: 3:47:43 PM
> > 
> > Config entry 'EMAIL ADDRESS' for roborock integration not ready yet: There are no devices that can currently be reached.; Retrying in background
> > ```
> 
> I had the same error after rolling back from 1.0.7.
> 
> I uninstall / reinstall the integration in HACS, reboot, and the configuration was good after that.

Interesting. That actually fixed it for me, working as expected here now.

**Comment by BoHome88** [comment]
Tried reinstalling. Still not connecting to the vacuum. Does it only work for US account holders?
